srcStatic/op2extStatic.vcxproj.filters

```diff
@@ -90,6 +87,9 @@
     <ClInclude Include="GameModules\EarthworkerProximityTasking.h">
       <Filter>GameModules</Filter>
     </ClInclude>
+    <ClInclude Include="Log\LogDestination.h">
```

Comment:
The path separator changed here too. Though there are other similar entries with the same slash.

Reply:
See response above. Likely done automatically by the IDE when I used it for edits.